README.md

```diff
@@ -1,3 +1,83 @@
+which contains the image descriptors with all modules that will be installed. In this repo you will find the basics 
+like which artifact (kieserver, business central, etc) is being used to build the openshift images. Let’s inspect the 
+rhdm-7-decisioncentral cekit module:
+
+
+```yaml
+schema_version: 1
+
+name: "rhdm-7-decisioncentral"
+description: "Red Hat Decision Manager Central 7.4 install"
+labels:
+    - name: "org.jboss.product"
+      value: "rhdm-decisioncentral"
+    - name: "org.jboss.product.version"
+      value: "7.4.0"
+    - name: "org.jboss.product.rhdm-decisioncentral.version"
+      value: "7.4.0"
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "rhdm-decisioncentral"
+    - name: "RHDM_DECISION_CENTRAL_VERSION"
+      value: "7.4.0"
+    - name: "PRODUCT_VERSION"
+      value: "7.4.0"
+    - name: "DECISION_CENTRAL_DISTRIBUTION_ZIP"
+      value: "DECISION_CENTRAL_DISTRIBUTION.ZIP"
+    - name: "DECISION_CENTRAL_DISTRIBUTION_EAP"
+      value: "jboss-eap-7.2"
+ports:
+    - value: 8001
+artifacts:
+    - name: DECISION_CENTRAL_DISTRIBUTION.ZIP
+      path: rhdm-7.4.0.DM-redhat-20190312-decision-central-eap7-deployable.zip
+      md5: 29e590113bf13f9c7c690473b523e444
+run:
+      user: 185
+      cmd:
+          - "/opt/eap/bin/standalone.sh"
+          - "-b"
+          - "0.0.0.0"
+          - "-c"
+          - "standalone.xml"
+execute:
+    - script: install
+```
+
+In the file above we set the most important configurations to which defines:
+    - the product
+    - the product version
+    - the product zip artifact, which will be deployed on the final OpenShift image.
+
+Note the *DECISION_CENTRAL_DISTRIBUTION.ZIP* env, its value will be the artifact name that, when the build is completed, 
+is placed in the `/<images_source_dir>/rhdm-7-openshift-image/businesscentral/target/image` directory, 
+note that it is on the rhdm-7-openshift-image repository. The **module.yaml** file above is a example of 
+the base module used to configure the product bits into the final OpenShift image. All OpenShift images have a 
+similar module descriptor, and these modules (the base modules) are one of the first module listed in
+the *-openshift image’s image.yaml file.
+ 
+
+This repo is used to build the final OpenShift images, they contain a set of yaml files, below you will find each 
+file and its purpose, all the modules have the same files, as described below:
+
+ - **container.yaml**: used by OSBS builds.
+ - **content_sets.yaml**: define the yum repositories needed to install dependencies for the image.
+ - **dev-overrides.yaml**: overrides file which uses the latest stable version for external dependencies but master for jboss-kie-modules and rhpam/rhdm-7-image.
+ - **rel-overrides.yaml**: overrides file which uses the latest stable version for all dependencies, i.e. cct_module, jboss-eap-modules, jboss-eap-image.
```

Comment:
- rel-overrides.yaml has been renamed to tag-overrides.yaml
- It is called *tag* overrides because it refers to the final tags that can be used to rebuild the image after release. For example, The 7.3.0.GA tag overrides in rhdm-7-openshift-image coincides with the rhdm-7.3.0.GA and rhpam-7.3.0.GA tags in jboss-kie-modules. This is intended for CVE respins.
- module.yaml and image.yaml files always refer to the latest versions of other repos (ie: master), which is why there are branch-overrides and tag-overrides files.